www/js/control/ProfileSettings.jsx

```diff
@@ -10,6 +10,10 @@ import DemographicsSettingRow from "./DemographicsSettingRow";
 import PopOpCode from "./PopOpCode";
 import ReminderTime from "./ReminderTime"
 import useAppConfig from "../useAppConfig";
+import AlertBar from "./AlertBar";
+import DataDatePicker from "./DataDatePicker";
+import AppStatusModal from "./AppStatusModal";
+import PrivacyPolicyModal from "./PrivacyPolicyModal";
```

Comment:
Addition of the `AlertBar` -- which shows a "snackbar" at the bottom of the screen with a message in replacement of many ionic alerts -> cases include text copied, data pushed, etc.
`DataDatePicker` used for choosing the data that json dump download is used for, replaces the ionic datepicker with react native paper's which is cleaner and works better
`AppStatusModal` and `PrivacyPolicyModal` we have talked about in the PR, so I'm sure you're up to speed on those, but were major changes here.